dlrover/python/elastic_agent/monitor/resource.py

```diff
@@ -60,7 +60,7 @@ def get_gpu_stats(gpus=[]):
     if not gpus:
         device_count = pynvml.nvmlDeviceGetCount()
         gpus = list(range(device_count))
-    gpu_stats_list = []
+    gpu_stats_list: list[GPUMetric] = []
```

Comment:
`gpu_stats_list` -> `gpu_stats`.  Generally, the variable name does not contain the data type.

Reply:
Ok, thanks for the correction. I'll keep an eye on this naming issue afterward as well.